easybuild/easyconfigs/l/LiBis/LiBis-20200428-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,54 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'LiBis'
+version = '20200428'
```

Comment:
better use the properly version source tarballs from https://pypi.org/project/LiBis?